src/main/kotlin/antlr/tree_format/TreeFormat.g4

```diff
@@ -0,0 +1,176 @@
+// Define a grammar called TreeFormat.
```

Comment:
@jpavlich  This is the updated grammar for Treepat. We are really concerned with the quality as we don't have much experience writing these type of code. Can you give use a review of the file and maybe give use some feedback regarding naming, format, readability, maintainability, etc.

We would also appreciate if you could review the TreeFormat.g4 file. 😁

Thank you! 😊